WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/variations/GenerateVariationPickerDialog.kt

```diff
@@ -0,0 +1,42 @@
+package com.woocommerce.android.ui.products.variations
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import com.woocommerce.android.R
+import com.woocommerce.android.databinding.DialogGenerateVariationsBinding
+
+class GenerateVariationPickerDialog(context: Context) : BottomSheetDialog(context) {
```

Comment:
⚙️ Can we inherit from `WCBottomSheetDialogFragment`? It contains some fixes for tablets + we'll be consistent with the rest of the app.
```suggestion
class GenerateVariationPickerDialog(context: Context) : WCBottomSheetDialogFragment() {
```

Reply:
> It contains some fixes for tablets + we'll be consistent with the rest of the app.

I believe we can add this enhancement directly to the view. The enhancement of `WCBottomSheetDialogFragment` was just to display the dialog fully expanded on tablets. I can work on this and come back to you later. I think It would be easy than using a Fragment. WDYT?

Reply:
I have to say I don't fully understand reasons for `WCBottomSheetDialogFragment`, even after reading the code comment.

I think this should be resolved by the Android framework instead 🤷  so I'm leaving the decision to you. I think we can skip this thing in this PR and validate if we need any changes as a separate issue.